### PR TITLE
Switch listView to recyclerView on NameChoosingFragment

### PR DESCRIPTION
**Pull request by alaaalzibda**

### CODE REVIEW

File: app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingAdapter.java [hunk 1]
```diff
@@ -137,55 +157,26 @@ int getNumInstances() {
...
         NameViewHolder(View view) {
+            super(view);
             ButterKnife.bind(this, view);
         }
 
-        void loadName(int position) {
-            this.position = position;
-            this.name.setText(currentState.getNameText(position));
+        void loadName() {
+            this.name.setText(currentState.getNameText(getAdapterPosition()));
```
Comment (Gear61) [inline app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingAdapter.java]: nit
```suggestion
            name.setText(currentState.getNameText(getAdapterPosition()));
```

File: app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
```
Comment (Gear61) [inline app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingFragment.java]: I assume this was for debugging. We can remove this now.

File: app/src/main/res/layout/name_choosing.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Gear61) [inline app/src/main/res/layout/name_choosing.xml]: nit: Slight formatting change
```suggestion
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
```

**Review by Gear61 — CHANGES_REQUESTED** [review]
This looks great and is 95% there, thanks for doing this!

Can you attach a video regression testing the list? Just add names to the list, make sure they show up, and try removing some items.

**Comment by alaaalzibda** [comment]
[A video shows recyclerView in the app](https://drive.google.com/open?id=1Ag5PGrT4Pc_xs-2MgV5gBMrpi9aYgkBQ)

**Pull request merged**